src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1291,7 +1291,7 @@ public String getValue() {
 
             @Override
             public String getDisplayName() {
-                return null;
+                return "";
```

Comment:
@amuniz We are trying to get a clean  `mvn clean verify -Djenkins.version=last_lts` build as well when upgrading to the parent POM, to anticipate possible PCT issues.